praw/models/reddit/multi.py

```diff
@@ -253,8 +251,5 @@ def update(
         path = API_PATH["multireddit_api"].format(
             multi=self.name, user=self._author.name
         )
-        response = self._reddit.request(
-            "PUT", path, data={"model": dumps(updated_settings)}
-        )
-        new = Multireddit(self._reddit, response["data"])
+        new = self._reddit.put(path, data={"model": dumps(updated_settings)})
```

Comment:
I just want to confirm that this properly objectifies `Multireddit`. It's definitely possible that we have a lapse in our test case for this.

Reply:
I tested this with `assert isinstance(new, Multireddit)` and it worked as expected. If you'd like, I can write a test that makes sure that this behavior is correct.

Reply:
That _should_ be sufficient.